src/sentry/tasks/post_process.py

```diff
@@ -178,7 +180,10 @@ def post_process_group(event, is_new, is_regression, is_new_group_environment, *
             # objects back and forth isn't super efficient
             for callback, futures in rp.apply():
                 has_alert = True
-                safe_execute(callback, event, futures)
+                with sentry_sdk.start_span(
```

Comment:
This was added in order to get the alert rule integrations/plugins included, I hope this is okay to add here but wasn't sure if there would be an issue. 
<img width="898" alt="Screen Shot 2020-03-26 at 3 16 12 PM" src="https://user-images.githubusercontent.com/15368179/77702022-c62dc800-6f74-11ea-88ea-eb9f312e646d.png">

cc @markstory @untitaker

Reply:
Do we have numbers on how many groups are created per second in prod? This seems somewhat dangerous to do, especially creating multiple transaction events for a single group.

Reply:
> Do we have numbers on how many groups are created per second in prod? 
I don't know the numbers on that, but won't this only create the transaction if they have alert rules set up? Maybe that's still too many transactions though I'm not sure. Maybe @JTCunning can weigh in here